### DISCUSSION TITLE
Chat Function

### DISCUSSION THREAD

**Discussion by Rocco-ks** [Traewelling/traewelling#3033] [Ideas / Feature Requests]
🇬🇧:Wouldn't it be nice if there was a chat function/PM function to chat with other users

🇩🇪: Wär es nicht nice wenn es eine Chat/Privatnachricht Funktion geben würde um mit anderen Usern zu schreiben?

**Comment by CitaroMH** [comment]
Hallo,

ja, ich fände eine Chatfunktion auch gut :)

Wir sind ja inzwischen schon eine sehr große Community.

**Comment by HerrLevin** [comment]
Eine Chatfunktion wäre zwar ganz angenehm zum Austausch, aber wir bekommen aktuell schon Meldungen über Belästigungen durch Statustexte. Eine Chat- und/oder Kommentarfunktion würde das wahrscheinlich ein gutes Stück nach oben Treiben und ebenso den damit verbundenen Moderationsaufwand. Ich glaube nicht, dass wir das aktuell stämmen könnten. 😕

**Reply by MrKrisKrisu** [reply]
Ich bin aus Gründen der Moderation auch aktuell eher dagegen, so interessant so ein Feature auch wäre.

**Reply by MrKrisKrisu** [reply]
Bzw. ergänzend dazu fände ich den Moderationsgedanken in einem „Chat” auch eher schlecht. Ich persönlich möchte nicht in fremden Chats „schnüffeln”, weil ein Teilnehmer diesen wegen Grund xy gemeldet hat. Das sehe ich in einen zu starken Eingriff. 

Ohne eine Melde- und Moderationsfunktion sollte es aber wiederum keine Chat oder Direktnachrichtfunktion geben.

Durchaus auch noch eine Zwickmühle.

**Reply by CitaroMH** [reply]
Also ich bin in diversen Foren aktiv und dort werden auch PNs geschrieben und die Moderation bekommt davon auch nichts mit, warum denn nicht auch hier?

**Reply by HerrLevin** [reply]
Foren haben leider eine ganz andere Sozialdynamik als Träwelling.

Viele Leute sehen diese Plattform als Konkurrenzkampf an, weil angeblich "andere Leute ihre Punkte durch fake check-ins" bekommen, was Leute gelegentlich zu öffentlichen Beleidigungen treibt. So etwas wollen wir nicht auf eine Chat- (oder Kommentar-)Ebene verlagern, da das einfach zu viel Potential auf weitere Ausschreitungen bietet, was mehr Moderationsaufwand mit sich bringt.

Weitere ehrenamtliche Moderator*innen können und wollen wir auch nicht als Gegenmaßnahme beschäftigen, da ein erweiterter Zugriff auf Profile von Usern – durch die Natur dieser Plattform – ein gewisses Missbrauchsrisiko mit sich birgt. Aus diesem Grund werden moderierende Tätigkeiten bei uns nur von Personen ausgeübt, die wir schon lange persönlich kennen.

Als alternative zur Kommunikation untereinander besteht die Überlegung, Profile mit einer Biografie und freiwilliger Verlinkung zu anderen Plattformen einzubinden. Das erhöht unseren Moderationsaufwand nur minimal, erweitert dennoch die Kommunikationsmöglichkeiten um einiges.

**Reply by CitaroMH** [reply]
Also das wäre auch eine sehr gute Idee.

**Reply by ChristianTacke** [reply]
Ich finde die Idee mit der Verlinkung anderer Plattformen gut. Dann kann man dort die bevorzugte Chat-Addr verlinken oder es lassen.

Keine Ahnung, ob man die dann nur für eingeloggte Nutzer sichtbar machen möchte, um Spam zu vermeiden?

Legt diese Funktion bitte allgemein genug an, damit man dort z.B. xmpp-URLs angeben kann.

**Reply by HerrLevin** [reply]
Mit #3495 in der API implementiert. Muss noch deployed werden (&evtl noch eine GUI gebaut werden :D)
